ovirt/resource_ovirt_cluster_test.go

```
@@ -18,17 +18,18 @@ import (
 func TestAccOvirtCluster_basic(t *testing.T) {
 	datacenterID := "5bc08e5b-03ab-0194-03cb-000000000289"
```

hardcoded datacenterID , failing with:
```**=== RUN   TestAccOvirtCluster_basic
    testing.go:654: Step 0 error: errors during apply:
        
        Error: Fault reason is "Operation Failed". Fault detail is "Entity not found: Datacenter: id=5bc08e5b-03ab-0194-03cb-000000000289". HTTP response code is "404". HTTP response message is "404 Not Found".
        
          on /tmp/tf-test131613786/main.tf line 2:
          (source code not available)
**```